fix build of sgEvolver on RHEL8 (without breaking build on CentOS 7)
/usr/include/bits/getopt_core.h:91:12: error: declaration of int getopt(int, char* const*, const char*) throw () has a different exception specifier
author: Kenneth Hoste (HPC-UGent)
--- sgEvolver/src/getopt.h.orig	2022-02-25 10:33:10.015721000 +0100
+++ sgEvolver/src/getopt.h	2022-02-25 10:33:33.950344000 +0100
@@ -146,7 +146,8 @@
 /* Many other libraries have conflicting prototypes for getopt, with
    differences in the consts, in stdlib.h.  To avoid compilation
    errors, only prototype getopt for the GNU C library.  */
-extern int getopt (int ___argc, char *const *___argv, const char *__shortopts);
+extern int getopt (int ___argc, char *const *___argv, const char *__shortopts)
+       __THROW __nonnull ((2, 3));
 # elif GETOPT_UNDEFINED /* not __GNU_LIBRARY__ */
 extern int getopt ();
 # endif /* __GNU_LIBRARY__ */
